src/Polly.Core/Simmy/Latency/LatencyConstants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Polly.Simmy.Latency;
+
+internal static class LatencyConstants
+{
+    public const string OnLatencyEvent = "OnLatency";
+
+    public static readonly TimeSpan DefaultLatency = TimeSpan.FromSeconds(30);
```

Comment:
Is this the current Simmy default? Just seems quite long to me 😃

Reply:
Yeah that was a suggestion from @martintmk as they use that default value internally at MS, open to suggestions tho